### ISSUE TITLE
Training loss and weights stuck

### ISSUE THREAD

**Issue by alipetiwala** [flairNLP/flair#705]
Following are the screenshots of my current training job:
![image](https://user-images.githubusercontent.com/41145154/57055453-2eb6eb00-6cb9-11e9-9842-db01c33158b4.png)

![image](https://user-images.githubusercontent.com/41145154/57055446-2494ec80-6cb9-11e9-9561-8a44d2764b27.png)

It seems  the training loss is saturated but the accuracy\F1 does show some activity. Also some of the weights like trainsitions are also saturated.  Should I stop training or continue till the full 150 epochs?

**Comment by alanakbik** [comment]
Hi @alipetiwala sorry for the late reply - have you found a solution for the problem? 

Generally, I would continue training as long as validation F1 goes up (which it still seems to do) and train loss goes down. If you use the standard settings, there is an annealing procedure in place that will anneal the learning rate if there are no improvements over a small number of epochs. So if you have the computational budget, I recommend running training until either trainings stops by itself because the annealing limit is reached, or until max_epochs is reached, which we generally set to a large number (depending on the problem/budget I like to set max_epochs to 100, 150 or even 300).
